src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java

```diff
@@ -1,10 +1,7 @@
 package de.tum.in.www1.artemis.web.rest;
 
 import java.io.IOException;
-import java.time.ZonedDateTime;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Teamscale finding for import. But: I'm not sure if we decided to use wildcards imports for >=4 or >=5 imports.

Reply:
Its still only 3 hier (ArrayList is no longer needed). Not sure why IntelliJ replaced it, I have it set to >= 4.